Mbaas/fix expiration time (#1201)
microsoft/appcenter-sdk-android: CHANGELOG.md

### App Center Data

* **[Feature]** Change the default time-to-live (TTL) from 1 day to infinite (never expire).

> * **[Fix]** Fix `isExpired` method in `LocalDocument` for incorrect handling `INFINITE` ttl situation. 

Reviewer: ```suggestion
* **[Fix]** Fix `isExpired` method in `LocalDocument` for incorrect handling of the `TimeToLive.INFINITE` value. 
```
